nucypher/blockchain/eth/token.py

```diff
@@ -205,9 +205,11 @@ def __init__(self,
 
         # Time
         self.start_datetime = datetime_at_period(period=first_locked_period,
-                                                 seconds_per_period=self.economics.seconds_per_period)
+                                                 seconds_per_period=self.economics.seconds_per_period,
+                                                 start_of_period=True)
```

Comment:
This is a bit confusing for a boolean flag. Would it be better as `is_start_of_period`?